javascript/ecmascript-2015/symbol/why-use-symbols-2.md

```diff
@@ -26,20 +26,20 @@ links:
 ---
 ## Content
 
-Probably the primary use of symbols is to introduce changes into EcmaScript. We can this in action with *iterators*.
+Probably the primary use of symbols is to introduce changes into EcmaScript. We can see this in action with `iterator`s.
 
-**Symbol.iterator** is used to identify something that can be iterated over. It is used by language features such as *for of* loops to understand how to work with an object.
+`Symbol.iterator` is used to identify something that can be iterated over. It is used by language features such as `for of` loops to understand how to work with an object.
 
 Don't worry too much about what the below code does just understand we are using a special inbuilt (referred to as well known) symbol to access the iterator functionality:
```

Comment:
hmmm I feel like "built-in" is more appropriate but 🤷‍♂ 	

https://english.stackexchange.com/questions/1250/built-in-or-in-built

Reply:
yeah I agree, "built-in" sounds much better

Reply:
done